renku_notebooks/api/schemas/version.py

```diff
@@ -5,7 +5,7 @@ class NotebooksServiceInfo(Schema):
     """Various notebooks service info."""
 
     anonymousSessionsEnabled = fields.Boolean(required=True)
-    cloudstorageEnabled = fields.Dict(required=True, keys=fields.String, values=fields.Boolean)
+    cloudstorageEnabled = fields.Boolean(required=True)
```

Comment:
Just a reminder - this requires a UI change. I think it is already handled but just in case.